win32ss/gdi/ntgdi/line.c

```diff
@@ -685,6 +680,78 @@ NtGdiPolyDraw(
     return result;
 }
 
+__kernel_entry
+W32KAPI
```

Comment:
We need that thing ("W32KAPI") as well? The implementation of the other entry points don't have it.

Reply:
No w don't need it here. Its just a 1:1 copy from the official prototype from MS headers. And removing it doesn't fix anything.